core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -328,11 +330,14 @@ oppia.directive('conversationSkin', [
 
           $scope.isLearnAgainButton = function() {
             var conceptCardIsBeingShown =
-              ExplorationEngineService.isStateShowingConceptCard();
+              ExplorationPlayerStateService.getCurrentEngineService().
```

Comment:
Same note about moving many of these functions to ExplorationPlayerStateService directly -- e.g. ExplorationPlayerStateService.isStateShowingConceptCard(). But, OK to file an issue to do this later as well.

Ditto below.

Reply:
Done.